seed/seedwriter/writer.go

```diff
@@ -631,6 +634,15 @@ func (w *Writer) Start(db asserts.RODatabase, f SeedAssertionFetcher) error {
 		return err
 	}
 
+	for _, extraAssertion := range w.opts.ExtraAssertions {
+		if err := f.Save(extraAssertion); err != nil {
+			return fmt.Errorf(
+				"cannot fetch and check prerequisites for an injected assertion: %v",
+				err,
+			)
+		}
+	}
```

Comment:
as I mentioned in the first review, especially because of how things are organized for UC20+ we don't want these in w.modelRefs but as separate